pages/04.applications/10.docs/noalyss/app_noalyss.fr.md

```diff
@@ -9,12 +9,12 @@ routes:
 
 Logiciel de comptabilité open source développé en PHP.
 
-Toute les infos sur ce logiciel sont disponible sur le [site du projet](http://www.noalyss.eu)
+Toute les infos sur ce logiciel sont disponible sur le [site du projet](http://www.noalyss.eu).
 
-Pour apprendre à l'utiliser je vous invite à lire le [wiki du projet](http://www.noalyss.eu/?page_id=46&lang=fr_FR)
+Pour apprendre à l'utiliser je vous invite à lire la [documentation utilisateur du projet](https://www.noalyss.eu/?page_id=1031).
 
-Le github du module YunoHost est [ici](https://github.com/YunoHost-Apps/noalyss_ynh)
+Le github du module YunoHost est [ici](https://github.com/YunoHost-Apps/noalyss_ynh).
```

Comment:
```suggestion
Le GitHub du module YunoHost est [ici](https://github.com/YunoHost-Apps/noalyss_ynh).
```

Reply:
Ooops already merged :/

Reply:
Solved in af9d3d3336aa4331c2f6cdf53657a894d700062e